awaitility/src/test/java/org/awaitility/AwaitilityTest.java

```diff
@@ -475,11 +475,11 @@ public Integer call() throws Exception {
     private abstract class AssertExceptionThrownInAnotherThreadButNeverCaughtByAnyThreadTest {
         public AssertExceptionThrownInAnotherThreadButNeverCaughtByAnyThreadTest() throws Exception {
             ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
-            System.setErr(new PrintStream(byteArrayOutputStream, true));
+            System.setErr(new PrintStream(byteArrayOutputStream, true, "UTF-8"));
```

Comment:
I missed it before, but I would use [`StandardCharsets.UTF_8`](https://docs.oracle.com/javase/7/docs/api/java/nio/charset/StandardCharsets.html#UTF_8) here.